### ISSUE TITLE
spaCy not running at all - macOS LLVM-gcc issue

### ISSUE THREAD

**Issue by TDaudert** [explosion/spaCy#2327]
Hi all,

I'm having an issue with running spaCy; I can't do anything basically. Everything was working smoothly until I updated my gcc yesterday. 
I'm always facing this error:

```
python3 -m spacy download en

Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/usr/local/lib/python3.6/site-packages/spacy/__init__.py", line 4, in <module>
    from .cli.info import info as cli_info
  File "/usr/local/lib/python3.6/site-packages/spacy/cli/__init__.py", line 6, in <module>
    from .train import train
  File "/usr/local/lib/python3.6/site-packages/spacy/cli/train.py", line 12, in <module>
    from ..gold import GoldCorpus, minibatch
ImportError: dlopen(/usr/local/lib/python3.6/site-packages/spacy/gold.cpython-36m-darwin.so, 2): Symbol not found: __ZdlPvm
  Referenced from: /usr/local/lib/python3.6/site-packages/spacy/gold.cpython-36m-darwin.so
  Expected in: /usr/lib/libstdc++.6.0.9.dylib
 in /usr/local/lib/python3.6/site-packages/spacy/gold.cpython-36m-darwin.so
```

## Your Environment
* Operating System: MacOS High Sierra 10.13.4
* Python Version Used:  3.6.5
* spaCy Version Used: 2.0.11 

I tried to run spaCy with multiple different settings and re-installed it as well as Python many times. 
For example, I used the native macOS LLVM as well as non-standard LLVMs:

- Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 9.1.0 (clang-902.0.39.1)
Target: x86_64-apple-darwin17.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
- gcc-4.9 (Homebrew GCC 4.9.4)
- gcc-7.2 (Homebrew GCC 7.2.0)
- gcc-8.1 (Homebrew GCC 8.1.0)
 
However, I cannot make it running. I hope you have some ideas how to solve this problem.
I came across this issue which seems to be related: [https://github.com/lanl/LaGriT/issues/107](https://github.com/lanl/LaGriT/issues/107)
Thanks in advance.

Best,
Tobias

**Comment by geyi** [comment]
I've had the same problem.

**Comment by ines** [comment]
We've just released `v2.0.13`, which comes with pre-build binary wheels for spaCy. We're also working on releasing wheels for all of our dependencies so users will be able to install spaCy without needing a compiler.

Could you try again installing the latest version via the wheels? If the issue still persists, feel free to reopen.
